stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/viewmodel/channel/ChannelListViewModel.kt

```diff
@@ -336,7 +336,7 @@ public class ChannelListViewModel(
     public fun deleteConversation(channel: Channel) {
```

Comment:
I feel like renaming this to `deleteChannel()` might be a good thing to do.
The same with `leaveGroup()` -> `leaveChannel()`.

I feel like this would cause less confusion for our users who have probably by this time gotten used to the idea of `Channels` and actively look for them when trying to perform actions. Since we're still in Beta this might be a good time to do this.

Reply:
Completely agree, let's not use the term "conversation" anywhere in our sources / docs.

Reply:
I'll raise an issue for the renaming and merge this one as is so we can have a more detailed renaming effort and not dirty up this PR.